azurerm/internal/services/compute/dedicated_host_group_data_source.go

```diff
@@ -61,7 +63,7 @@ func dataSourceArmDedicatedHostGroupRead(d *schema.ResourceData, meta interface{
 	name := d.Get("name").(string)
 	resourceGroupName := d.Get("resource_group_name").(string)
 
-	resp, err := client.Get(ctx, resourceGroupName, name)
+	resp, err := client.Get(ctx, resourceGroupName, name, compute.InstanceView)
```

Comment:
To keep consistency with the old api-version, could we use an empty string for this new parameter?
```suggestion
	resp, err := client.Get(ctx, resourceGroupName, name, "")
```